.kokoro/release/stage.sh

```diff
@@ -33,7 +33,6 @@ retry_with_backoff 3 10 \
   mvn clean deploy -B \
     --settings ${MAVEN_SETTINGS_FILE} \
     -DskipTests=true \
-    -Dclirr.skip=true \
```

Comment:
this should be adding this line to the 2.1.x branch here: https://github.com/googleapis/java-bigtable-hbase/blob/2.1.x/.kokoro/release/stage.sh#L34